lib/spanner/command/reqrep.ex

```diff
@@ -57,12 +48,10 @@ defmodule Spanner.Command.Request do
 
   defp read_config(request, config_path) do
     [bundle, _cmd] = String.split(request.command, ":")
-    cmd_config_file = Path.join([config_path, bundle, "config.json"])
+    cmd_config_file = Path.join([config_path, bundle, Config.dynamic_file_name()])
```

Comment:
Adding a module which wraps two hard-coded attributes is overkill. Move the attributes to `reqrep.ex` and drop `Config`, imo.